srcpkgs/gitless/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'gitless'
+pkgname=gitless
+version=0.8.6
+revision=1
+noarch=yes
+build_style=python3-module
+pycompile_module="$pkgname"
+hostmakedepends="python3-setuptools"
+makedepends="python3-devel"
+depends="git>=1.7.12 python3-pygit2 python3-sh python3-clint"
```

Comment:
i don't think depending on specific version here is necessary, 1.7.12 is extremely old that it is unlikely someone has it